backend/src/openarchiefbeheer/destruction/tests/endpoints/test_abort.py

```diff
@@ -88,6 +87,63 @@ def test_cannot_abort_without_comment(self):
         self.assertEqual(response.status_code, status.HTTP_400_BAD_REQUEST)
         self.assertEqual(response.json()["comment"][0], _("This field is required."))
 
+    def test_abort_list_process(self):
+        record_manager = UserFactory.create(
+            username="record_manager", post__can_start_destruction=True
+        )
+        record_manager_group, created = Group.objects.get_or_create(
+            name="Record Manager"
+        )
+        administrator_group, created = Group.objects.get_or_create(name="Administrator")
+        record_manager.groups.add(record_manager_group)
+        record_manager.groups.add(administrator_group)
+        destruction_list = DestructionListFactory.create(
+            name="A test list",
+            author=record_manager,
+            status=ListStatus.ready_to_review,
+            planned_destruction_date=date(2024, 1, 8),
```

Comment:
The planned_destruction_date is usually only present on lists with status ready_to_delete (but i guess it is to check that it gets cleared all in one test).

Reply:
TBH: I've just renamed/altered the existing test, but now you mention it, it is useful in a test setting. We could add another test if you think thats's better?

Reply:
My bad :sweat_smile: I think we can leave it but maybe with a comment?

Reply:
I'll update the test for the process, this behavior is tested explicitly in the test for the destruction state